include/core.hpp

```diff
@@ -33,7 +33,10 @@ namespace color_coded
 
     inline std::string temp_dir()
     {
-      static auto const dir(fs::temp_directory_path() / "color_coded/");
+      static auto const temp_dir(fs::temp_directory_path());
+      static auto const user_dir(std::string("color_coded-") +
+                                 std::to_string(geteuid()));
```

Comment:
Since I am not an avid C++ programmer, I am confused between the following options:

```cpp
static auto const dir(temp_dir / user_dir / "");
```

and:

```cpp
static auto const user_dir(std::string("color_coded-") +
                           std::to_string(geteuid()) + std::string("/"));
```

How would you have done it, and why?